bin/validate-data-package

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Script: validate_frictionless_package.sh
+# Description: Bash script to validate a Frictionless Data Package using the Frictionless CLI.
+# Usage: validate_frictionless_package.sh [-v tides_version | -l local_schema_location] [-d dataset_location]
+#   -v tides_version: Optional. Specify the version of the TIDES specification or 'local' to
```

Comment:
Is this parameter necessary? Shouldn't this always use the schema specified in the datapackage or treat `-l` as an override? In other words, if `-l` is provided, override the datapackage, otherwise use the datapackage, no need for `-v`.

Reply:
In fact, it seems like `-l` should be `-s`, which lets you set any schema location, overriding any datapackage. It wouldn't need to be local, could also be a GitHub location or NFS or samba, &c.

Reply:
Also, the samples/template/TIDES doesn't validate, should it?